Shared/Samples/List spatial reference transformations/ListSpatialReferenceTransformationsView.swift

```diff
@@ -0,0 +1,174 @@
+                    } catch {
+                        self.error = error
+                    }
+                }
+                .errorAlert(presentingError: $error)
+            
+            NavigationView {
+                TransformationsList(model: model)
+                    .navigationTitle("Transformations")
+                    .navigationBarTitleDisplayMode(.inline)
+                    .toolbar {
+                        ToolbarItem(placement: .topBarTrailing) {
+                            transformationsMenu
+                        }
+                    }
+            }
+            .navigationViewStyle(.stack)
+        }
+    }
+    
+    /// A menu containing actions relating to the list of transformations.
+    private var transformationsMenu: some View {
+        Menu("Transformations", systemImage: "ellipsis") {
+            Picker("Filter Transformations", selection: $filterByMapExtent) {
+                Label("All Transformations", systemImage: "square.grid.2x2")
+                    .tag(false)
+                Label("Suitable for Extent", systemImage: "line.3.horizontal.decrease.circle")
+                    .tag(true)
+            }
+            .pickerStyle(.inline)
+            .onChange(of: filterByMapExtent) { newValue in
+                model.updateTransformationsList(withExtent: newValue ? visibleArea?.extent : nil)
+            }
+            
+            Link(destination: .projectionEngineDataDownloads) {
+                Label("Download Data", systemImage: "arrow.down.circle")
+            }
+            
+            Button("Set Data Directory", systemImage: "folder") {
+                fileImporterIsPresented = true
+            }
+        }
+        .fileImporter(
+            isPresented: $fileImporterIsPresented,
+            allowedContentTypes: [.folder]
+        ) { result in
+            do {
+                switch result {
+                case .success(let fileURL):
+                    try model.setProjectionEngineDataURL(fileURL)
+                case .failure(let error):
+                    throw error
+                }
+            } catch {
+                self.error = error
+            }
+        }
+    }
+}
+
+private extension ListSpatialReferenceTransformationsView {
+    struct TransformationsList: View {
+        /// The view model for the sample.
+        @ObservedObject var model: Model
+        
+        /// The missing Projection Engine filenames for the tapped transformation.
+        @State private var missingFilenames: [String] = []
+        
+        var body: some View {
+            List(model.transformations, id: \.self) { transformation in
+                Button {
+                    if transformation.isMissingProjectionEngineFiles {
+                        missingFilenames = model.missingProjectionEngineFilenames(
+                            for: transformation
+                        )
+                        model.removeSelection()
```

Comment:
```suggestion
```
If the transformation with missing projection file isn't selected, no need to de-select it?